patchwise/main.py

```diff
@@ -19,11 +20,19 @@
 from .patch_review.ai_review.ai_review import add_ai_arguments, apply_ai_args
 from .patch_review.kernel_tree import create_git_worktree
 from .patch_review.patch_review import PATCH_PATH
+from .curse_setup import curse_pipeline
+from patchwise import PACKAGE_PATH
```

Comment:
Is this supposed to be:
```suggestion
from . import PACKAGE_PATH
```
@Andrei-Treil, can you whether to use `from patchwise` or `from .` in this case?

Reply:
PEP8 says absolute import paths and relative imports from siblings are equally valid, so it's up to personal preference (running Ruff on both versions keeps whatever is already there). Personally, I prefer  `from patchwise import PACKAGE_PATH` since I wouldn't immediately think to check the init.py file if the import was `from . import PACKAGE_PATH `